src/main/java/core/basesyntax/Entry.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Entry<K, V> {
+    private final K key;
+    private V value;
+
+    public Entry(K key, V value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    public V getValue() {
+        return value;
+    }
+
+    public void setValue(V value) {
+        this.value = value;
+    }
+
+    public K getKey() {
+        return key;
+    }
+
```

Comment:
According to the checklist, getters and setters should not be used in the `Entry` class as it's an inner class and we have direct access to its private fields. Consider removing `getValue()`, `setValue()`, and `getKey()` methods.